README.md

```diff
@@ -1,3 +1,20 @@
 # Moqui AWS Integrations
 
 Amazon Web Services tool integrations for Moqui Framework
+
+* To be able to work with aws you should have an amazon user. <br>
+For secure purposes create IAM user with administrator access instead of using just your root user. <br>
+More information about creating a user you can find [here](https://docs.aws.amazon.com/mediapackage/latest/ug/setting-up-create-iam-user.html)
```

Comment:
Revision: "More information about creating a user can be found [here](https://docs.aws.amazon.com/mediapackage/latest/ug/setting-up-create-iam-user.html)."